reference/api/settings.mdx

```diff
@@ -1910,3 +1911,189 @@ Reset an index's typo tolerance settings to their [default value](#typo-toleranc
+| **`inputField`**       | Array of strings | []                                                                       | Path to location of document data in query Meilisearch sends to embedder                                                                                       |
+| **`inputType`**        | String           | `text`                                                                   | Type of data Meilisearch sends to embedder. Must be `text` or `textArray`                                                                                      |
+| **`query`**            | String           | Object                                                                   | Extra fields Meilisearch adds to queries Meilisearch sends to embedder                                                                                         |
+| **`pathToEmbeddings`** | Array            | []                                                                       | Path to vector embedding data in embedder response                                                                                                             |
+| **`embeddingObject`**  | String           | {}                                                                       | Name of the embedding object in embedder response                                                                                                              |
+| **`distribution`**     | Object           | Empty                                                                    | Describes the natural distribution of search results. Must contain two fields, `mean` and `sigma`, each containing a numeric value between `0` and `1`         |
+
+Depending on your chosen embedder, some of these fields may be mandatory.
+
+### Get embedder settings
+
+<RouteHighlighter method="GET" route="/indexes/{index_uid}/settings/embedders"/>
+
+Get the embedders configured for an index.
+
+#### Path parameters
+
+| Name              | Type   | Description                                                               |
+| :---------------- | :----- | :------------------------------------------------------------------------ |
+| **`index_uid`** * | String | [`uid`](/learn/core_concepts/indexes#index-uid) of the requested index |
+
+#### Example
+
+<CodeSamples id="get_embedders_1" />
+
+##### Response: `200 OK`
+
+```json
+{
+  "default": {
+    "source":  "openAi",
+    "apiKey": "OPENAI_API_KEY",
+    "model": "text-embedding-3-small",
+    "documentTemplate": "A movie titled {{doc.title}} whose description starts with {{doc.overview|truncatewords: 20}}",
+    "dimensions": 1536
+  }
+}
+```
+
+### Update embedder settings
+
+<RouteHighlighter method="PATCH" route="/indexes/{index_uid}/settings/embedders"/>
+
+Partially update the embedder settings for an index. When this setting is updated Meilisearch may reindex all documents and regenerate their embeddings.
```

Comment:
I feel like at some point we should specify **when** Meilisearch will reindex everything, but this is currently in flux for v1.9 so we might want to wait a bit to do this (maybe opening an issue in the docs?).

Reply:
You mean specifically for vector search stuff, or in general? Either way, yes, an issue might be the best approach. 

Here you go: #2831

Reply:
I meant specifically for vector search stuff, but you're right that with the settings diff of v1.9 it would be better to have this in general.

Loving #2831, I think we can resolve this comment and continue the discussion there at a later date